### PR TITLE
Release Notes 99

### PR DESCRIPTION
**Pull request by meagancojocar**
## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.
- [ ] If making css changes, I rebuilt the bundle.

### CODE REVIEW

File: themes/default/content/blog/pulumi-release-notes-94/index.md
Comment (arunkumar611) [inline themes/default/content/blog/pulumi-release-notes-94/index.md:1]: `make lint`seems to have reordered the table of contents based on the order in the blog. Should be okay?

File: themes/default/content/blog/pulumi-release-notes-99/index.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+
+
+## Core
+
+Placeholder
+
+## Providers and Packages
+
+### AWS 6.0 to AWSX
```
Comment (meagancojocar) [inline themes/default/content/blog/pulumi-release-notes-99/index.md:98]:
```suggestion
### AWSX now uses AWS 6.0
```

File: themes/default/content/blog/pulumi-release-notes-99/index.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  - [Pulumi ESC Preview](#pulumi-esc-preview)
+  - [Developer Portal Gallery](#developer-portal-gallery)
+  - [Deployments GA](#deployments-ga)
+  - [Customer Managed Agents](#customer-managed-agents)
+  - [MFA](#mfa)
+  - [Pricing Calculator](#pricing-calculator)
+  - [Historical Views](#historical-views)
+- [Core](#core)
+- [Providers and Packages](#providers-and-packages)
+  - [AWS 6.0 to AWSX](#aws-60-to-awsx)
```
Comment (meagancojocar) [inline themes/default/content/blog/pulumi-release-notes-99/index.md:33]:
```suggestion
  - [AWSX now uses AWS 6.0](#awsx-now-uses-aws-60)
```

File: themes/default/content/blog/pulumi-release-notes-99/index.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+Placeholder
+
+### New Community Providers
+
+Placeholder
+
+## Wrap up
+
+That's a wrap on our latest Pulumi release notes! With each new feature and improvement, from AI advancements to enhanced cloud deployment tools, we're excited to see how you leverage these updates. Your [feedback](https://github.com/pulumi/pulumi-cloud-requests/issues/new/choose) is crucial in shaping our path forward. tay connected for more updates, and here's to building a more efficient cloud future together!
```
Comment (meagancojocar) [inline themes/default/content/blog/pulumi-release-notes-99/index.md:122]:
```suggestion
That's a wrap on our latest Pulumi release notes! With each new feature and improvement, from AI advancements to enhanced cloud deployment tools, we're excited to see how you leverage these updates. Your feedback is crucial in shaping our path forward, feel free to open issues in our [Pulumi Cloud requests repository](https://github.com/pulumi/pulumi-cloud-requests/issues/new/choose)  or in the [pulumi/pulumi repository](https://github.com/pulumi/pulumi) for CLI feedback. Stay connected for more updates, and here's to building a more efficient cloud future together!
```

**Review by meagancojocar — COMMENTED** [review]
LTGM!

**Comment by justinvp** [comment]
@arunkumar611, I added a commit with a note about Import Improvements. I also have some small updates to the import docs in https://github.com/pulumi/pulumi-hugo/pull/3893. Would be great to get that merged before the blog post is published.

**Review by arunkumar611 — APPROVED** [review]
Approving since it's been reviewed.

File: themes/default/content/blog/pulumi-release-notes-99/index.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+{{< video title="Pulumi new AI and deploy" src="../pulumi-ai-new/pulumi-new-ai.mp4" controls="false" autoplay="true" loop="true" >}}
+
+### Pulumi AI coverage and performance
+
+Pulumi AI can now write Pulumi programs for all 150 cloud providers in the [Pulumi registry](/registry/) - up from 20 cloud providers. Pulumi AI has also been updated to OpenAI's GPT-4 Turbo for enhanced performance along with fine tuning of our prompts to improve our AI's ability to write code.
+
+### Pulumi AI Answers
+
+Pulumi AI Answers has become a go-to repository, hosting an extensive collection of over 50,000 anonymized, infrastructure-as-code commonly asked queries, all carefully selected and presented by Pulumi. As this repository continues to expand, we're seeing an exponential increase in user traffic, underscoring the immense value it offers. It's inspiring to see the Pulumi community actively leveraging it as they build and scale their cloud infrastructure
```
Comment (meagancojocar) [inline themes/default/content/blog/pulumi-release-notes-99/index.md:56]:
```suggestion
[Pulumi AI Answers](https://www.pulumi.com/ai/answers), launch late last year, has become a go-to repository, hosting an extensive collection of over 50,000 anonymized, infrastructure-as-code commonly asked queries, all carefully selected and presented by Pulumi. As this repository continues to expand, we're seeing an exponential increase in user traffic, underscoring the immense value it offers. It's inspiring to see the Pulumi community actively leveraging it as they build and scale their cloud infrastructure
```

**Pull request merged**